--- croner-3.0.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ croner-3.0.1/Cargo.toml	2025-11-27T20:57:08.148995+00:00
@@ -19,7 +19,7 @@
 autoexamples = false
 autotests = false
 autobenches = false
-description = "Fully-featured, lightweight, and efficient Rust library designed for parsing and evaluating cron patterns"
+description = "Rust library designed for parsing and evaluating cron patterns"
 homepage = "https://github.com/hexagon/croner-rust"
 documentation = "https://docs.rs/croner"
 readme = "README.md"
